components/opponent/commons/opponent.lua

```diff
@@ -254,6 +255,9 @@ function Opponent.resolve(opponent, date, options)
 			else
 				PlayerExt.populatePageName(player)
 			end
+			if not player.team and options.syncPlayer then
+				PlayerExt.syncTeam(player.pageName:gsub(' ', '_'), nil)
```

Comment:
```suggestion
				player.team = PlayerExt.syncTeam(player.pageName:gsub(' ', '_'), nil)
```

Reply:
should I keep the PrizePool/Base file (in this same PR) or I can just delete it if that one is not required a change anymore?